src/android/CDVBroadcaster.java

```diff
@@ -107,6 +109,22 @@ private void fireNativeEvent( final String eventName, JSONObject userData ) {
         sendBroadcast( intent );
     }
 
+    private String sendGlobalBroadcast(final String eventName, final String actionName, JSONObject userData) {
+        if( eventName == null ) {
+            throw new IllegalArgumentException("eventName parameter is null!");
+        }
+
+        Log.w("eventName: ", eventName);
+        Log.w("actionName: ", actionName);
+
+        final Intent intent = new Intent(eventName)
+                .setAction(actionName)
+                .putExtras(toBundle(new Bundle(), userData));
+
+        this.webView.getContext().sendBroadcast(intent);
+        return intent.toString();
+    }
```

Comment:
what's the benefit/value in returning `intent.toString()`? Just noticing that `fireNativeEvent()` doesn't return anything

Reply:
Oops, that was me debugging.